scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -1177,9 +1177,14 @@ <h4 class="modal-title" id="filterNameModalLabel">Filter display name</h4>
   {{ pin_indicator(variant, case) }}
 {% endmacro %}
 
-{% macro pagination_hidden_div(page, total_variants) %}
+{% macro pagination_hidden_div(page, result_size) %}
```

Comment:
I renamed these to make the macros more consistent with the callers nomenclature.